api/src/main/resources/workbench.yaml

```diff
@@ -2201,6 +2201,15 @@ definitions:
         type: "array"
         items:
           $ref: "#/definitions/ParticipantCohortStatus"
+      page:
+        type: integer
+      pageSize:
+        type: integer
+      sortOrder:
+        type: string
+      sortColumn:
+        type: string
+
```

Comment:
Can we make these names match the query parameters on the getParticipantCohortStatuses api definition? Notice here: https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/workbench.yaml#L836
we're using the following names to define the pagination variables:
1) page
2) limit
3) order
4) column
I don't really care which we use. I only care that they are consistent.

Reply:
I went with `page, pageSize, sortOrder, sortColumn` - I find it a little bit more explicit.

Reply:
fine by me.